WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/MainState.kt

```diff
@@ -241,30 +213,18 @@ private fun speakerIsSupported(activity: Activity): Boolean {
     val hasBuiltInSpeaker = devices.any { it.type == AudioDeviceInfo.TYPE_BUILTIN_SPEAKER } &&
         hasAudioOutputFeature
 
-    val hasBluetoothSpeaker = devices.any { it.type == AudioDeviceInfo.TYPE_BLUETOOTH_A2DP }
+    // Check all the Wear supported BT devices
```

Comment:
This function could live in horologist if it's reusable and documented.

Reply:
Currently Horologist inherits this logic via media3.